openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -27,6 +27,29 @@ def escape_space(path):
     return '"{}"'.format(path) if " " in path else path
 
 
+def get_ocio_config_path(profile_folder):
```

Comment:
@iLLiCiTiT I wonder if this should not be somewhere in global scope?

Reply:
I would create new `openpype.lib` module `colorspace.py` and add it there.

Reply:
We've discussed this on discord and conclusion was that it is right now used only here and is used for very specific usecase (fix maketx bug). Handling ocio configs properly would require much more logic and having it in `openpype.lib` would maybe cause illusion of us handling the colorspace and using them which is not true.

Reply:
I think we could just move it elsewhere if we need to, soon as we decide to tackle the color management.